### ISSUE TITLE
AsyncStreamSession.cpp No member named 'thisThreadInQueue' 

### ISSUE THREAD

**Issue by KCastiglione** [jgh-/VideoCore#225]
in latest version 24645b6508e7f15c1ab9c6f3c07048ff606067f6 there are 10 errors in this file: AsyncStreamSession.cpp

     AsyncStreamSession.cpp:235:29: No member named 'thisThreadInQueue' in 'videocore::JobQueue'
     AsyncStreamSession.cpp:249:28: No member named 'thisThreadInQueue' in 'videocore::JobQueue'
     AsyncStreamSession.cpp:260:28: No member named 'thisThreadInQueue' in 'videocore::JobQueue'
     AsyncStreamSession.cpp:269:28: No member named 'thisThreadInQueue' in 'videocore::JobQueue'

**Comment by KCastiglione** [comment]
@dourgulf looks like from the latest RTMP parser

**Comment by tonyisid** [comment]
I have the same problem

**Comment by KCastiglione** [comment]
@tonyisid yes it is from the latest version, just pod install, and no changes.

@jgh- @dourgulf please let us know if any fixes or suggestions. thank you!

**Comment by tonyisid** [comment]
yes, wish some suggestions for fix this problem. Thanks very much.

**Comment by tonyisid** [comment]
yes, wish some suggestions for fix this problem. Thanks very much.

**Comment by maxcampolo** [comment]
Yes @dourgulf we would appreciate some feedback on this as it is your update that added these asserts. What is the `thisThreadInQueue()` method supposed to check for, that the job is the current job? It is not present in `JobQueue.hpp`. I'm not sure it's ideal to rely on these asserts running anyway. Was this code for debugging and supposed to be taken out?

For anyone with this problem, I fixed this in my branch by just taking out these lines for now.

**Comment by dourgulf** [comment]
Just remove AsyncStreamSession.cpp from your project please. It's a deprecated code.

**Comment by dourgulf** [comment]
I have make a pull request to fixed this bug.

**Comment by KCastiglione** [comment]
@dourgulf thanks for the pull request and update 


@jgh- can you please accept the pull request, thank you once again for a great library

**Comment by ppamorim** [comment]
Same here
